src/commands/general/Stats.ts

```diff
@@ -147,10 +147,14 @@ export class Stats extends BaseCommand {
             }
         };
 
-        if (ctx.guild && !showAll) {
-            const pts = await LoggerManager.getPoints((resMember as IResolvedMember).member);
+        const pts = await LoggerManager.getPoints((resMember as IResolvedMember).member);
+        let description = `${EmojiConstants.TICKET_EMOJI} Points: ${pts}`;
+        if(stats.quotaPoints > 0) description +=`\nQuota Points: ${stats.quotaPoints}`;
```

Comment:
Run VSCode's Format Document feature on all files you create/change so we can have somewhat consistent style.

Reply:
Format document led to typescript errors that prevented compilation.

Reply:
What error? That's interesting lol

Reply:
Also the screenshot is misleading, the formatting is correct based on the slight logic adjustment.